Library/Formula/docbook.rb

```diff
@@ -43,9 +43,6 @@ class Docbook < Formula
   end
 
   def install
-    (etc/"xml").mkpath
-    system "xmlcatalog", "--noout", "--create", "#{etc}/xml/catalog"
```

Comment:
Might be nice to still create this in install inside `buildpath` and then use `etc.install`